commit d8f5fe8f3f83be53c76146f3858c4745ae261d55
Author: Thomas Harning Jr
Date:   Fri Jul 25 16:49:14 2008 -0400

    libusb build fixed - configuration happens according to Apple's build process and shared libraries are deleted after-build

diff --git a/libusb/Makefile b/libusb/Makefile
index 50f5eb3..474eed1 100644
--- a/libusb/Makefile
+++ b/libusb/Makefile
@@ -1,7 +1,7 @@
 Project               = libusb
 UserType              = Administrator
 ToolType              = Libraries
-GnuAfterInstall       = install-plist remove-autom4te-cache
+GnuAfterInstall       = install-plist remove-autom4te-cache remove-shared-libusb
 Extra_Configure_Flags = 
 
 include $(MAKEFILEPATH)/CoreOS/ReleaseControl/GNUSource.make
@@ -14,7 +14,7 @@ AEP_Version    = 0.1.13b
 AEP_ProjVers   = $(AEP_Project)-$(AEP_Version)
 AEP_Filename   = $(AEP_ProjVers).tar.gz
 AEP_ExtractDir = $(AEP_ProjVers)
-AEP_Patches    = #configure.diff gcc42.diff
+AEP_Patches    = skipConfigure.patch FixConfigureAddCPP.patch #configure.diff gcc42.diff
 #svn co -r658 https://libusb.svn.sourceforge.net/svnroot/libusb/trunk
 
 # Extract the source.
@@ -25,11 +25,8 @@ install_source::
 	for patchfile in $(AEP_Patches); do \
 		(cd $(SRCROOT)/$(Project) && patch -p0 < $(SRCROOT)/files/$$patchfile) || exit 1; \
 	done
-	(cd $(SRCROOT)/$(Project) && \
-	./autogen.sh --enable-shared=no --disable-dependency-tracking \
-	 CFLAGS="-arch ppc -arch i386" LDFLAGS="-arch ppc -arch i386" CXXFLAGS="-arch ppc -arch i386" )
-	$(RMDIR) $(SRCROOT)/$(Project)/autom4te.cache
-#	$(RM) $(SRCROOT)/$(Project)/config.status
+
+	(cd $(SRCROOT)/$(Project) && ./autogen.sh --skip-configure )
 	
 OSV = $(DSTROOT)/usr/local/OpenSourceVersions
 OSL = $(DSTROOT)/usr/local/OpenSourceLicenses
@@ -41,4 +38,7 @@ install-plist:
 	$(INSTALL_FILE) $(Sources)/LICENSE $(OSL)/$(Project).txt
 
 remove-autom4te-cache:
-	rm -rf $(SRCROOT)/$(Project)/autom4te.cache
\ No newline at end of file
+	rm -rf $(SRCROOT)/$(Project)/autom4te.cache
+
+remove-shared-libusb:
+	rm -f $(DSTROOT)/usr/lib/libusb*{dylib,la}
diff --git a/libusb/files/skipConfigure.patch b/libusb/files/skipConfigure.patch
new file mode 100644
index 0000000..57c8798
--- /dev/null
+++ b/libusb/files/skipConfigure.patch
@@ -0,0 +1,13 @@
+--- autogen.sh-old	2008-07-25 16:43:41.000000000 -0400
++++ autogen.sh	2008-07-25 16:44:53.000000000 -0400
+@@ -59,6 +59,10 @@
+ autoconf${AUTOCONF_SUFFIX}
+ cd $ORIGDIR
+ 
++if [ "$1" == "--skip-configure" ]; then
++	exit
++fi
++
+ $srcdir/configure --enable-maintainer-mode "$@" || exit
+ 
+ echo 
